### ISSUE TITLE
Accessibility Testing for VA on MHV, Medical Records, Imaging

### ISSUE THREAD

**Issue by BobbyBaileyRB** [department-of-veterans-affairs/va.gov-team#98846]
### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

BobbyBaileyRB

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### Who completed the axe scans?

BobbyBaileyRB

### axe checks

- [x] Each page has been scanned using axe (results shared below)
- [x] axe is integrated into your end-to-end testing

### axe DevTools scan results

![Image](https://github.com/user-attachments/assets/48e2d90e-41b5-4e71-8559-82a353c3f2c4)

![Image](https://github.com/user-attachments/assets/bb7b7fe7-af86-4709-b7c9-de03379d25d3)

### Who completed the content zoom and reflow test?

BobbyBaileyRB

### Content zoom and reflow checks

- [x] All page elements are readable and usable at 200% zoom
- [x] All page elements are readable and usable at 300% zoom
- [x] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

When zoomed to 200-400%, focus states do not navigate to the corresponding content as expected when moving between pages.

### Who completed the keyboard navigation test?

BobbyBaileyRB

### Keyboard navigation checks

- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [x] All elements under focus have a visible focus indicator
- [x] The order of [Tab] stops made sense and was appropriate for completing tasks

### Do you have any other results to share?

**Title tags**
The title tags for pages are not unique, as both the detail page and the image page share the same meta title: KNEE 4 OR MORE VIEWS (LEFT). It would be more appropriate for the image page to use a distinct title, such as Images: KNEE 4 OR MORE VIEWS (LEFT).

**Alt tags**
- The alt text for images is not meaningful, likely due to the lack of metadata. Currently, images are labeled as alt="1 - 01/01". A more effective approach might be to use descriptive alt text like "Image 1, details not provided" for better clarity and accessibility. 
- These also have Aria labels on them of aria-label="4" which could be useful if not providing the number order within the alt text
- Be aware some images have the alt-text of alt="4 - 1842/01" which I am not sure if this is coming from a database. But with the aria-label it's confusing and complexed "4, 4 - 1842/01".
- It could also be helpful to provide guidance for screen reader folks on where to find these details if they are not available on the image pages.

**Browser back arrow**
Using the back arrow does not return the view to the focused element or header.

**Comment by bryan-riley-va** [comment]
I've created MHV-65022 to address
